homeassistant/components/osoenergy/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "osoenergy",
+  "name": "OSO Energy",
+  "codeowners": ["@osohotwateriot"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/osoenergy",
+  "iot_class": "cloud_polling",
+  "requirements": ["pyosoenergyapi==1.0.4"]
```

Comment:
In `pyosoenergyapi` you specify `requests` as a dependency, but you don't use it. Please remove it

Reply:
I've removed the unused dependencies